src/ufonormalizer.py

```diff
@@ -1090,6 +1090,8 @@ def _convertPlistElementToObject(element):
             else:
                 obj[key] = _convertPlistElementToObject(subElement)
     elif tag == "string":
+        if not element.text:
+            return ""
         return xmlEscapeText(element.text)
```

Comment:
or maybe you could fix it in `xmlEscapeText` directly, ensuring that the latter never returns None, but always at least an empty string. Looking at the other places that function is used, it seems like the expectation is that it takes a string and returns some other modified string, not None.

Reply:
A case for mypy!

Reply:
Hm. Making xmlEscapeText always return a string makes another testcase fail
```
        writer = XMLWriter(declaration=None)
        writer.propertyListObject({None: ""})
        self.assertEqual(
            writer.getText(),
>           '<dict>\n\t<key/>\n\t<string></string>\n</dict>')
E       AssertionError: '<dict>\n\t<key></key>\n\t<string></string>\n</dict>' != '<dict>\n\t<key/>\n\t<string></string>\n</dict>'
E         <dict>
E       - 	<key></key>
E       ? 	    -- ---
E       + 	<key/>
E         	<string></string>
E         </dict>
```